polyfill/test/intl.mjs

```diff
@@ -4,14 +4,16 @@ const { describe, it, report } = Demitasse;
 import Pretty from '@pipobscure/demitasse-pretty';
 const { reporter } = Pretty;
 
-import { strict as assert } from 'assert';
-const { deepEqual, equal, throws } = assert;
+import { strict as Assert } from 'assert';
+const { deepEqual, equal, ok: assert, throws } = Assert;
```

Comment:
I think you'll want to revert this.

Reply:
I don't understand, what's wrong here?

Reply:
We just made them all consistent in #448, why should this one be inconsistent?